packages/core/adder/config.ts

```diff
@@ -29,6 +29,7 @@ export type Adder<Args extends OptionDefinition> = {
 	id: string;
 	alias?: string;
 	environments: Environments;
+	oneOrTwoWordDescription?: string;
```

Comment:
```suggestion
	shortDescription?: string;
```

Reply:
I had thought about calling is `shortDescription`, but I thought it was better to be extra explicit about the expected length for folks building community adders

Reply:
I don't think we need to be this explicit, as we are always going to review at least the official ones. If we are going to have something similar for community adders is another discussion. 

Another alternative could be `keyword`

Reply:
i prefer `shortDescription` as well. we have full control of this so we can certainly limit ourselves

Reply:
Yeah, I'm not worried about the official ones since we control those. We don't have any control over the community adders though and aren't they using the same interface?

https://github.com/sveltejs/cli/blob/e4c41d650a6031ded725d4fade7415a961358a3b/community-adder-template/src/index.js#L13

Reply:
They do, but I don't think we'll be displaying their descriptions anywhere in the CLI if we're only going invoke them via args:
```shell
sv add tailwindcss some-community-add-on another-one

# skips over the selection prompt

# ...add-on specific questions and so on
```

Reply:
Hmm. I don't trust that people wouldn't fill it out with something long. We should perhaps have a different interface for them to use if we're not going to use the field. Or maybe have some validation that only shows it if it's short. Anyway, I can rename it for now